### PR TITLE
Feature/Delete Endpoint

### PR DESCRIPTION
**Pull request by EmmanuelChukwujindu**
This adds delete endpoint

### CODE REVIEW

File: PR/PRAPI/Controllers/WeatherForecastController.cs [hunk 1]
```diff
@@ -35,5 +35,12 @@ public IEnumerable<WeatherForecast> Get()
             })
             .ToArray();
         }
+
+        [HttpDelete]
```
Comment (Endurance-Peter) [inline PR/PRAPI/Controllers/WeatherForecastController.cs:39]: why are we using [HttpDelete] without user id

**Review by Endurance-Peter — CHANGES_REQUESTED** [review]

File: PR/PRAPI/Controllers/WeatherForecastController.cs (hunk 1 above)
Comment (EmmanuelChukwujindu) [reply PR/PRAPI/Controllers/WeatherForecastController.cs:39]: oh yes, we want to abide by rest specifications

File: PR/PRAPI/Controllers/WeatherForecastController.cs [hunk 2]
```diff
@@ -35,5 +35,12 @@ public IEnumerable<WeatherForecast> Get()
...
+        public IActionResult Delete()
+        {
+            return Ok();
+        }
+
```
Comment (BenjaminSunday710) [inline PR/PRAPI/Controllers/WeatherForecastController.cs:44]: Nice work....Weldone

**Review by BenjaminSunday710 — APPROVED** [review]

File: PR/PRAPI/Controllers/WeatherForecastController.cs (hunk 1 above)
Comment (Endurance-Peter) [reply PR/PRAPI/Controllers/WeatherForecastController.cs:39]: use post instead

**Review by Endurance-Peter — APPROVED** [review]

**Review by BenjaminSunday710 — APPROVED** [review]

**Pull request merged**